docs/data-management.rst

```diff
@@ -167,9 +167,15 @@ For HCS data, you can create new Screens and link Plates to these Screens.
 |image2e|
 
 .. note::
-    If you are an administrator or administrator with restricted privileges working in a group you are not a member of: All containers (Projects, Datasets, Screens) created in OMERO.web in such group will belong to the user whose portfolio you are working with. This helps to retain the possibility for this user to manipulate their containers even though you created them.
+    Organizing data of other users as an administrator, restricted administrator or group owner is made easier in OMERO.web in following manner.
```

Comment:
in *the* following manner

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/11/commits/bd60ff8c763f009dd89970843550b47e60e0dd21